bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-toolbar/component.jsx

```diff
@@ -283,23 +283,21 @@ class WhiteboardToolbar extends Component {
      * 4. Trigger initial animation for the icons
     */
     // 1st case
-    if (this.thicknessListIconRadius && this.thicknessListIconColor) {
```

Comment:
The displayed code change appears quite complicated but I have simply removed the "if (this.thicknessListIconRadius && this.thicknessListIconColor) " condition.